enginetest/queries.go

```diff
@@ -8569,6 +8569,81 @@ var errorQueries = []QueryErrorTest{
 		Query:       `SELECT * FROM datetime_table where datetime_col >= 'not a valid datetime'`,
 		ExpectedErr: sql.ErrConvertingToTime,
 	},
+	{
+		Query: `SELECT t1.*
+					  FROM
+						mytable as t1,
+						mytable as t2,
+						mytable as t3,
+						mytable as t4,
+						mytable as t5,
+						mytable as t6,
+						mytable as t7,
+						mytable as t8,
+						mytable as t9,
+						mytable as t10,
+						mytable as t11,
+						mytable as t12,
+						mytable as t13,
+						mytable as t14,
+						mytable as t15
+					  WHERE
+						t1.i = t2.i and
+						t2.i = t3.i and
+						t3.i = t4.i and
+						t4.i = t5.i and
+						t5.i = t6.i and
+						t6.i = t7.i and
+						t7.i = t8.i and
+						t8.i = t9.i and
+						t9.i = t10.i and
+						t10.i = t11.i and
+						t11.i = t12.i and
+						t12.i = t13.i and
+						t13.i = t14.i and
+						t14.i = t15.i`,
+		ExpectedErr: sql.ErrUnsupportedJoinCount,
+	},
+	{
+		Query: `SELECT t1.*
+					  FROM
+						mytable as t1,
+						mytable as t2,
+						mytable as t3,
+						mytable as t4,
+						mytable as t5,
+						mytable as t6,
+						mytable as t7,
+						mytable as t8,
+						mytable as t9,
+						mytable as t10,
+						mytable as t11,
+						mytable as t12,
+						mytable as t13,
+						mytable as t14,
+						mytable as t15`,
+		ExpectedErr: sql.ErrUnsupportedJoinCount,
+	},
+	{
+		Query: `SELECT t1.*
```

Comment:
Make sure we don't error out in analysis when joining e.g. 2 subqueries that have 10 tables each

Reply:
yeah that's a good point, didn't think of that. i guess opaque nodes should be included in the outer scope table count, but have their children counted independently.

Reply:
on second though, inserting subqueries in a join doesn't affect the join complexity, right? ex: a 3 table join with subqueries inserted is still a 3 table join. same is true for 20 table join.
```SQL
> select * from (select * from a) as a, (select * from b) as b , (select * from c) as c;
```